po/fr.po

```diff
@@ -238,6 +238,381 @@ msgstr "Pacman est en cours d'utilisation, veuillez patienter..."
+"jour du système"
+
+#: src/help.rs:66
+msgid ""
+"    --[no]combinedupgrade  Refresh then perform the repo and AUR upgrade "
+"together"
+msgstr ""
+"    --[no]combinedupgrade  Rafraîchit et effectue les mises à jour du dépôt "
+"et de l'AUR en même temps"
+
+#: src/help.rs:67
+msgid ""
+"    --[no]batchinstall     Build multiple AUR packages then install them "
+"together"
+msgstr ""
+"    --[no]batchinstall     Compile plusieurs paquets depuis l'AUR et les "
+"installe en même temps"
+
+#: src/help.rs:68
+msgid ""
+"    --[no]provides         Look for matching providers when searching for "
+"packages"
+msgstr ""
+"    --[no]provides         Recherche des fournisseurs correspondants lors de "
+"la recherche des paquets"
+
+#: src/help.rs:69
+#, fuzzy
+msgid "    --[no]devel            Check development packages during sysupgrade"
+msgstr "    --[no]devel            Vérifie"
+
+#: src/help.rs:70
+msgid ""
+"    --[no]installdebug     Also install debug packages when a package "
+"provides them"
+msgstr ""
+"    --[no]installdebug     Installe aussi les paquets de déboguage quand un "
+"paquet les fournit"
+
+#: src/help.rs:71
+msgid ""
+"    --[no]sudoloop         Loop sudo calls in the background to avoid timeout"
+msgstr ""
+"    --[no]sudoloop         Appelle sudo en arrière plan pour éviter "
+"l’expiration du délai"
+
+#: src/help.rs:72
+msgid "    --[no]chroot           Build packages in a chroot"
+msgstr "    --[no]chroot           Compile les paquets dans un chroot"
+
+#: src/help.rs:73
+msgid "    --[no]sign             Sign packages with gpg"
+msgstr "    --[no]sign             Signe les paquets avec gpg"
+
+#: src/help.rs:74
+msgid "    --[no]signdb           Sign databases with gpg"
+msgstr "    --[no]signdb           Signe les bases de données avec gpg"
+
+#: src/help.rs:75
+msgid "    --localrepo            Build packages into a local repo"
+msgstr "    --localrepo            Compile les paquets dans un dépôt local"
+
+#: src/help.rs:76
+#, fuzzy
+msgid ""
+"    --nocheck              Don't resolve checkdepends or run the check "
+"function"
+msgstr ""
+"    --nocheck              Ne pas résoudre les dépendances ou lancer la "
+"fonction de vérification"
+
+#: src/help.rs:77
+msgid ""
+"    --develsuffixes        Suffixes used to decide if a package is a devel "
+"package"
+msgstr ""
+"    --develsuffixes        Suffixes utilisés pour décider si un paquet est "
+"un paquet de développement"
+
+#: src/help.rs:78
+msgid ""
+"    --bottomup             Shows AUR's packages first and then repository's"
+msgstr ""
+"    --bottomup             Afficher les paquets de l'AUR en premier et ceux "
+"du dépôt ensuite"
+
+#: src/help.rs:79
+msgid ""
+"    --topdown              Shows repository's packages first and then AUR's"
+msgstr ""
+"    --topdown              Affiche les paquets du dépôt et ceux de l'AUR "
+"ensuite"
+
+#: src/help.rs:81
+msgid "show specific options:"
+msgstr "affiche les options spécifiques :"
```

Comment:
show is lowercase because it's the name of the flag `--show`. Seems as this is changed to a normal word it should be capitalised.